index.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio Hendrid</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+    <header>
+        <a href="/" class="header-logo">
+            <h3>
+                Welcome
+            </h3>
+        </a>
+        <nav>
+            <img src="./assest/Union.svg" alt="menu link">
+        </nav>
+    </header>
+    <section class="headline">
+        <h2>
+            Hey there. I’m Hendrid
+        </h2>
+        <h2>
+            <span>I’m a software developer</span>
+        </h2>
+        <p>
+            I can help you build a product , feature or website Look through some of my work and experience! If you like
+            what you see and have a project you need coded,
+            don’t hestiate to contact me.
+        </p>
+        <div class="social-media">
```

Comment:
- [ ] [REQUIRED] I think the list of icons uses a ul tag to group them, and li per icon; for each icon that has a link (a tag). This is used to present list of information in well formed and semantic way.